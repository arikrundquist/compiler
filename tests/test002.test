
/*

test handling of escape sequences

*/

actor test2 {

    str = "testing |\t|\r|\n|\"|\'|\\| escape sequences"

}
