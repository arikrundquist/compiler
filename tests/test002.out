RESERVED	actor
IDENTIFIER	test2
OPERATOR	{
IDENTIFIER	str
OPERATOR	=
STRING	OBJECT	testing |	||
|"|'|\| escape sequences
OPERATOR	}
